docs/hub/keras.md

```diff
@@ -1,75 +1,56 @@
 # Using Keras at Hugging Face
 
-`keras` is an open-source machine learning library that uses a consistent and simple API to build models leveraging TensorFlow and its ecosystem.
+Keras is an open-source multi-backend deep learning framework, with support for JAX, TensorFlow, and PyTorch. You can find more details about it on [keras.io](https://keras.io/).
 
 ## Exploring Keras in the Hub
 
 You can find over 200 `keras` models by filtering at the left of the [models page](https://huggingface.co/models?library=keras&sort=downloads).
```

Comment:
```suggestion
You can find over 1200 `keras` models by filtering at the left of the [models page](https://huggingface.co/models?library=keras&sort=downloads).
```

smol suggestion: https://huggingface.co/models?library=keras&sort=downloads
(or we should change the tag if these are not the ones we want to showcase).

Reply:
Thanks @Vaibhavs10! Actually this is indeed the link we want to share but it is currently polluted by Keras 2.x models ([see this list](https://huggingface.co/models?library=keras,tf-keras&sort=downloads)). I'm in the process of opening PRs to fix these but maybe we just shouldn't provide a fixed number in the docs. I'll review this, thanks for spotting it!

Reply:
addressed in https://github.com/huggingface/hub-docs/pull/1320/commits/9a754c09df38fb8dc639ef123f14351a4454b0bc